frontend/src/components/models.ts

```diff
@@ -71,7 +71,7 @@ export const supportedChains: Array<Chain> = [
     logoURI: ETH_NETWORK_LOGO,
   },
   {
```

Comment:
*arbitrary comment location*

Sorting by date received is broken on the receive table for Gnosis Chain only (it seems).

Sends are sorted with oldest first (should be newest), and the table order does not react to clicking the "Date Received" header.

We've [seen this before](https://github.com/ScopeLift/umbra-protocol/issues/316) but couldn't pin it down. Is this the same thing?

Reply:
<img width="978" alt="Screen Shot 2023-06-13 at 3 33 56 PM" src="https://github.com/ScopeLift/umbra-protocol/assets/1479802/9e57f25a-97bd-4987-8175-7ecc3dd875f9">

Reply:
Sorting by Date Received is fix by https://github.com/ScopeLift/umbra-protocol/pull/536/commits/b17ee653db20221a7f169c22aedfdbefbe21852b
Having trouble reproducing wrong sort result, but added additional sorting https://github.com/ScopeLift/umbra-protocol/pull/536/commits/e27b2cd2c250a722cef8a8cab2c3d069c0651b8e to try to prevent it from occurring too often